applications/RomApplication/custom_utilities/rom_residuals_utility.h

```diff
@@ -72,6 +72,18 @@ namespace Kratos
 
         ~RomResidualsUtility()= default;
 
+        /**
+         * Resizes a Matrix if it's not the right size
+         */
+        template<typename TMatrix>
+        static void ResizeIfNeeded(TMatrix& rMat, const std::size_t Rows, const std::size_t Cols)
+
+        {
+            if(rMat.size1() != Rows || rMat.size2() != Cols) {
+                rMat.resize(Rows, Cols, false);
+            }
+        };
+
```

Comment:
repeated code in RomBuilderAndSolver. We should add use it onle once (probably in a separate PR)

Reply:
Adding to the TODO list.